logprep/processor/list_comparison/processor.py

```diff
@@ -97,8 +97,7 @@ def add_rules_from_directory(
                 f"({current_process().name})"
             )
         self.ps.setup_rules(
-            [None] * self._generic_tree.rule_counter
-            + [None] * self._specific_tree.rule_counter
+            [None] * self._generic_tree.rule_counter + [None] * self._specific_tree.rule_counter
```

Comment:
why are we taking back the format change?

Reply:
black does his thing. And 100 chars are 100 chars. :-)
your black configuration does not respect the pyproject.toml in project root. you have to fix it.